### ISSUE TITLE
Templates listed in a new namespace under Run button look like they're from default not the current namespace

### ISSUE THREAD

**Issue by rawlingsj** [hawtio/hawtio-kubernetes#105]
After creating a new namespace (`test`) and using gofabric8 to install console and latest templates into the new namespace I can see the templates are correct after running 

```
oc export templates --namespace test
```

Using the new console in the `test` namespace, I navigate to the `test` team and click `Run`, select the cd-pipeline template and `Next`, I expand an RC that's about to be run and can see it's not the same template that's installed in the current `test` namespace.  I've searched the other namespaces and think the templates displayed and run by the UI are in fact coming from the `default` namespace.

So it looks like the console is using templates from default and not the current namespace.

**Comment by gashcrumb** [comment]
Yup, added this last week, felt less kludgy than having to flip over to
'default' to deploy stuff into a different namespace.

**Comment by rawlingsj** [comment]
Ah I see.. ok in this scenario the new namespace has the latest version of templates which I want to run rather than the old ones in the default namespace.

**Comment by gashcrumb** [comment]
Maybe I could add a checkbox to show/hide the ones in 'default'?  Might be a nice way to handle either scenario...

**Comment by rawlingsj** [comment]
Yeah I think that would do nicely
